src/app/components/query-explainer/query-explainer.component.html

```diff
@@ -0,0 +1,232 @@
+<ng-container *ngIf="componentData$ | async as componentData">
+    <app-verify-result-section [verifyResponse]="componentData.sqlQueryVerify" />
+
+    <ng-container *ngIf="componentData.sqlQueryExplainerResponse">
+        <div class="row">
+            <div class="col-md-6">
+                <div class="box p-4 my-2 ms-4 h-100">
+                    <div class="position-relative border border-1 p-2 rounded-3">
+                        <span class="position-absolute text-muted group-box-label text-uppercase">Input data</span>
+                        <div class="mt-4 mx-4">
+                            <div class="fw-semibold">Query:</div>
```

Comment:
I think we can and should extract more components from this.
A proposal:
 - Input data section
 - Commitment data section

These would be very easy to test separately, as they become simply a display panels without much logic.

Reply:
Created more components